### PR TITLE
manolo tlc ciel d8 8_2b to 8_3

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Ciel/Day 8/08_01_CIEL08_2B.txt [hunk 1]
```diff
@@ -6,12 +6,12 @@ Fourth period ends.
...
 -- Page 10006, Offset 18697.
 -- 茶道室に行ってシエル先輩に会うか、ひとりで済ませるか。
-Should I go visit Ciel-senpai at the tea ceremony room? Or should I
+Should I visit Ciel-senpai at the tea ceremony room? Or should I eat lunch alone?
 }
 [sha:6e3a98a02e3fc66eab6e6ba54f0e55a52d8853a2]{
 -- Page 10006, Offset 18698.
 -- どちらにしても購買部で昼食を買ってからの話だ。
-Whichever choice I make will come after I've bought my lunch at the cafeteria.
+Anyway, that'll have to wait until after I buy some lunch at the cafeteria.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_2B.txt:14]: in the cafeteria

File: script/Ciel/Day 8/08_01_CIEL08_2B.txt [hunk 2]
```diff
@@ -26,15 +26,15 @@ Senpai?"
 [sha:caa4d1175102d5da722c4fd2badd12ebae653317]{
 -- Page 169, Offset 18701.
 -- 購買部に向かう途中、中庭を横切っていく先輩の姿が見えた。
-As I was heading for the cafeteria, I see Senpai cutting across the inner courtyard.
+On the way to the cafeteria, I catch a glimpse of Senpai crossing the courtyard.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_2B.txt:29]: on the way, I catch

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 3]
```diff
@@ -1,27 +1,29 @@
 [sha:7cc854c20b617d1112edc84dfdacd9a2394486fa]{
 -- Page 10007, Offset 18704.
 -- ……怪しい。あの購買部好き……というか特定のパン好き……の先輩が、わざわざ外にランチしに行くとは思えない。
-...Very suspicious. I don't think that my school store loving, or rather special bread loving, Senpai would deliberately go out to have lunch.
+// god save me
+...Suspicious. My cafeteria loving Senpai, or rather, my specifically one kind of bread loving Senpai wouldn't go out of her way to have lunch somewhere outside school.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_2_1.txt:5]: ...Suspicious. My cafeteria-loving Senpai—well, %{i}bread%{/i}-loving Senpai—wouldn't go out of her way to eat lunch outside

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 4]
```diff
@@ -1,27 +1,29 @@
...
 }
 [sha:25602e2a6c002d27c6e9c4ba96d260a4a3904683]{
 -- Page 172, Offset 18705.
 -- もしや、保健室で言っていた“吸血鬼のねぐら”を調査しに行くのではないだろうか？
-Isn't she rather going to investigate that vampire roost she talked about in the infirmary, perchance?
+// was ねぐら=roost decided on?
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_2_1.txt:10]: I hope not. "roost" is pretty terrible. In day 7 we used "nest" which I think is much better language. The Church especially uses "vermin-like" language to describe vampires and vampirings

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 5]
```diff
@@ -1,27 +1,29 @@
...
 -- Page 172, Offset 18706.
 -- だとしたら放っておけない。先輩にいくつか確認したい事もある。
-If so, then I can't leave her alone. There are also a few things I want to ask Senpai.
+If so, then I can't let her go alone. There are also a couple things I need to clear up with her.
 }
 [sha:0737fee4a0a22f3fa05918b9cdd356fec5667bb5]{
 -- Page 173, Offset 18707.
 -- 「……それに、アルクェイドとの約束もあるし。吸血鬼のねぐらを事前に見つけられるなら、アイツだって喜ぶだろう」
-"...Besides, I also have my promise with Arcueid. If we find where the roost is ahead of time, I'm sure she'll be happy."
+"...Besides, there's also my promise to Arcueid. If we manage to locate the roost beforehand, I'm sure she'll be happy."
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_2_1.txt:21]: nest

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 6]
```diff
@@ -41,7 +43,7 @@ I hurriedly leave through the main gate.
 [sha:7f289df940224765799e423fc2878336836fbb74]{
 -- Page 175, Offset 18712.
 -- こっちは全力で走って、先輩は普通に中庭を歩いていたものの、やはり距離の差があった。
-I'm running at full speed, whereas Senpai is walking at a normal pace through the inner courtyard, but we're separated by a good distance.
+Though I was running at full speed while Senpai was walking at a normal pace, as expected, there was still some distance between us.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_2_1.txt:46]: pace, there was still the sort of distance one would expect between us.

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 7]
```diff
@@ -56,7 +58,8 @@ Senpai is...
 [sha:34e24dc3cf6b0e6b057bcd170e2ed93abda652fe]{
 -- Page 176, Offset 18715.
 -- いた、駅方面の通学路だ。
-There, on the road leading to school but heading toward the station.
+// school road? road I take going to school? road students commute on?
+There, on the school route. She was headed in the direction of the station.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_2_1.txt:62]: the road from the station to the school

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt [hunk 8]
```diff
@@ -811,12 +812,13 @@ She lets out a scorning laugh aimed at herself and turns around.
...
 -- Page 237, Offset 18865.
 -- 昨夜そういう内容のホラー映画を借りてしまって。あんまりにも真に迫っていたので、遠野くんにもホラーを分けてあげようかなと」
-That's the plot of a horror movie I borrowed yesterday night. It felt so incredibly real that I wanted to share that horror with you, Tohno-kun."
+It's the plot of a horror movie I rented yesterday night. It felt so incredibly real that I wanted to share that horror with you, Tohno-kun."
 }
 [sha:70722b4ab998f0c55529b0adbe5e30869a5b0f7b]{
 -- Page 238, Offset 18866.
 -- 「そ―――そうですか。とつぜん意地が悪くなったコトはどうかと思いますけど、その映画自体は面白そうですね。二重構造でよく出来ています。タイトルを教えてくれれば俺も借りてみますよ」
-"I-Is that so? I don't know what to think of you making fun of me, but the movie itself seems to be interesting. The embedded narrative is nicely done. Tell me the title and I'll borrow it too."
+// embedded narrative sounds kinda weird but might just be me. change to something like "meta" maybe?
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_2_1.txt:820]: meta-narrative

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 8/08_01_CIEL08_2_1.txt (hunk 4 above)
Comment (nolo16) [reply script/Ciel/Day 8/08_01_CIEL08_2_1.txt:10]: originally had this as nest but changed it back. i'll go ahead and change the other instances of 'roost' in d8 then

**Pull request merged**